modules/gcp_bigquery_big_lake_table/inputs.tf

```diff
@@ -59,9 +67,11 @@ variable "source_format" {
     condition     = contains(["NEWLINE_DELIMITED_JSON", "AVRO", "PARQUET"], var.source_format)
     error_message = "Source format of table must be NEWLINE_DELIMITED_JSON, AVRO or PARQUET"
   }
+  default = "PARQUET"
 }
 
 variable "source_uris" {
   type        = list(string)
   description = "A list of the fully-qualified URIs that point to your data in Google Cloud. https://cloud.google.com/bigquery/docs/external-data-cloud-storage#wildcard-support"
+  default     = null
```

Comment:
type says list(string) whereas the default values is null.

Reply:
It can default to `[]`, updated.